tests/test_unit/test_cdf_tk/test_commands/test_dump_data.py

```diff
@@ -95,6 +99,69 @@ def test_dump_assets(self, tmp_path: Path) -> None:
         parquet_files = list(parquet_dir.rglob("*.parquet"))
         assert len(parquet_files) == 1
 
+    def test_dump_filemetadata(self, tmp_path: Path) -> None:
```

Comment:
Maybe I couldn't find it, but I would like to see some tests for some non-happy paths edge cases such as:

- Test case with some "bad" files in it (invalid MIME, special chars in name/external Id etc)
- what happens if metadata is empty?